sqlc-gen-ftl/src/plugin/mod.rs

```diff
@@ -0,0 +1,187 @@
+#![allow(dead_code)]
```

Comment:
this is to bypass warns when running the test - everything is actually used in the WASM build but appears unused to the test analyzer since we test through sqlc. every function declaration would need to be annotated if we don't do it at the module level (can do this if you find it preferable, but i felt it looked a bit noisy)